drivers/timer/stm32_lptim_timer.c

```diff
@@ -86,6 +86,11 @@ static struct k_spinlock lock;
 #endif
 #endif /* !CONFIG_STM32_LPTIM_TICK_FREQ_RATIO_OVERRIDE */
 
+static inline bool ARRM_STATE_GET(void)
+{
+	return ((LL_LPTIM_IsActiveFlag_ARRM(LPTIM) != 0) && LL_LPTIM_IsEnabledIT_ARRM(LPTIM));
```

Comment:
Nit: Both functions return the same type of information (uint32_t State of bit (1 or 0)), so please be consistent: 
```suggestion
	return (LL_LPTIM_IsActiveFlag_ARRM(LPTIM) && LL_LPTIM_IsEnabledIT_ARRM(LPTIM));
```
```suggestion
	return ((LL_LPTIM_IsActiveFlag_ARRM(LPTIM) != 0) && (LL_LPTIM_IsEnabledIT_ARRM(LPTIM) != 0));
```